python/pyspark/streaming/kinesis.pyi

```diff
@@ -32,15 +35,15 @@ class KinesisUtils:
         regionName,
         initialPositionInStream,
         checkpointInterval,
-        storageLevel: Any = ...,
-        awsAccessKeyId: Optional[Any] = ...,
-        awsSecretKey: Optional[Any] = ...,
-        decoder: Any = ...,
-        stsAssumeRoleArn: Optional[Any] = ...,
-        stsSessionName: Optional[Any] = ...,
-        stsExternalId: Optional[Any] = ...,
+        storageLevel: StorageLevel = ...,
+        awsAccessKeyId: Optional[str] = ...,
+        awsSecretKey: Optional[str] = ...,
+        decoder: Callable[[T], T] = ...,
+        stsAssumeRoleArn: Optional[str] = ...,
+        stsSessionName: Optional[str] = ...,
+        stsExternalId: Optional[str] = ...,
     ): ...
```

Comment:
The output is missing. I believe it should be `DStream[T]`:

```python
) -> DStream[T]: ...
```

Reply:
Added it, thanks!